modules/renter/persist_test.go

```diff
@@ -142,7 +142,9 @@ func TestRenterSaveLoad(t *testing.T) {
 	rt.renter.saveFile(f3)
 
 	// load should now load the files into memory.
+	id := rt.renter.mu.Lock()
 	err = rt.renter.load()
+	rt.renter.mu.Unlock(id)
```

Comment:
hmm. In general, I try to avoid using non-exported functions and fields when testing something, unless it's a unit test. I think it helps with the design and overall coverage.

(that's for making changes - for reading the state and verifying it has the expected values, I'll use unexported fields and functions)